src/Plugin/HelpSection/ProfileHelpMaintainingSection.php

```diff
@@ -0,0 +1,81 @@
+<?php
+
+namespace Drupal\stanford_profile\Plugin\HelpSection;
+
+use Drupal\help\Plugin\HelpSection\HelpSectionPluginBase;
+
+/**
+ * Provides the module topics list section for the help page.
+ *
+ * @HelpSection(
+ *   id = "profile_help_maintaining",
+ *   title = @Translation("Maintaining your site"),
+ *   description = @Translation("Resources for maintaining the health of your site."),
+ *   weight = -99
+ * )
+ */
+class ProfileHelpMaintainingSection extends HelpSectionPluginBase {
+
+  /**
+   * {@inheritdoc}
+   */
+  public function listTopics() {
+    return [
+      $this->getMaintainingYourSite(),
+      $this->getStanfordWebServicesSupport(),
+      $this->getSiteHealth(),
+      $this->getContent(),
+    ];
+  }
+
+  /**
+   * Get the user guide help text.
+   *
+   * @return array
+   *   Markup render array.
+   */
+  protected function getMaintainingYourSite() {
```

Comment:
@mdyoung3 I was hoping this would actually be a level up, as the overarching h2.  This could replace the current  intro to that section.